nipype/interfaces/afni/preprocess.py

```diff
@@ -2492,6 +2492,14 @@ class TProjectInputSpec(AFNICommandInputSpec):
                    even if -ort contains constant terms, as all means are
                    removed.""",
         argstr="-polort %d")
+    dsort = File(
+        desc="""Remove the 3D+time time series in dataset fset.
+                ++ That is, 'fset' contains a different nuisance time
+                series for each voxel (e.g., from AnatICOR).
+                ++ Multiple -dsort options are allowed.""",
```

Comment:
Since multiple `-dsort` options are allowed, let's allow multiple files.

Here's an example, but you'll want to use `InputMultiObject` instead of `InputMultiPath`

https://github.com/nipy/nipype/blob/704b97dee7848283692bac38f04541c5af2a87b5/nipype/interfaces/dcm2nii.py#L342-L344